src/actions/create-comment.ts

```
@@ -0,0 +1,41 @@
+"use server";
+
+import { COMMENT_POST } from "@/functions/api";
+import { revalidateTag } from "next/cache";
+import { cookies } from "next/headers";
+import { apiError } from "@/functions/api-error";
+import { ActionResponse, Comment } from "@/shared/types";
+
+export async function createComment(
+  state: {},
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)